src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,863 @@
+            </ul>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>"plugin"</inlineCode></p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>network</inlineCode></td>
+        <td><inlineCode>NetworkPermission</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to access the network.</p>
+            <p>For example, to make HTTP requests (XHR, fetch, etc.), load images (<inlineCode>&lt;img src="" /&gt;</inlineCode>), etc.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no network access)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>webview</inlineCode></td>
+        <td><inlineCode>WebviewPermission</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to use webviews in its UI to display web content or complex UI.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no webview usage)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>launchProcess</inlineCode></td>
+        <td><inlineCode>LaunchProcessPermission</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to launch processes using APIs like <inlineCode>require("uxp").shell.openPath()</inlineCode> or <inlineCode>shell.openExternal()</inlineCode>.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no process launching)</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### NetworkPermission
+
+<p>Specifies the domains that the plugin can access in network requests.</p>
+<p><strong>Example</strong></p>
+<code class="language-json">{`{
+    "domains": [
+        "https://example.com",
+        "https://*.adobe.com/",
+        "wss://*.myplugin.com"
+    ]
+}
+`}</code>
+<p>Then, in your plugin code, you can make network requests like this:</p>
+<code class="language-javascript">{`const response = await fetch("https://example.com");
+`}</code>
+<p>Or load images like this:</p>
+<code class="language-html">&lt;img src="https://example.com/image.png" /&gt;
+</code>
+<p>You can also allow access to all domains by setting <inlineCode>domains</inlineCode> to <inlineCode>"all"</inlineCode>.</p>
+<code class="language-json">{`{
+    "domains": "all"
+}
+`}</code>
+<h4>Properties</h4>
+<table>
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><inlineCode>domains</inlineCode> *</td>
+        <td><inlineCode>string[] | "all"</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The domains that the plugin can access. Can be a list of domains, or the string "all" to allow access to all domains.</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### WebviewPermission
```

Comment:
WebView instead of webview

Reply:
I dont have a recipe for this topic but we should mention this link https://developer-stage.adobe.com/indesign/uxp/reference/uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLWebViewElement/ for docs.